hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlan.java

```diff
@@ -134,6 +147,30 @@
         return plans;
     }
 
+    private static Address[] initPartitionOwnersAndMembers(NodeEngine nodeEngine, Collection<Member> members) {
```

Comment:
This method modifies the given member collection too, not just returns the partition owners.